src/test/java/com/artipie/asto/FileStorageTest.java

```diff
@@ -93,6 +94,20 @@ public void savesAndLoads() throws Exception {
         );
     }
 
+    @Test
+    public void saveOverwrites() throws Exception {
+        final byte[] original = "1".getBytes();
+        final byte[] updated = "2".getBytes();
+        Assume.assumeThat(updated, Matchers.not(Matchers.equalTo(original)));
+        final BlockingStorage storage = new BlockingStorage(
+            new FileStorage(this.folder.newFolder().toPath())
+        );
+        final Key key = new Key.From("foo");
+        storage.save(key, original);
+        storage.save(key, updated);
+        MatcherAssert.assertThat(storage.value(key), Matchers.equalTo(updated));
```

Comment:
@olegmoz Do not use static matchers, and put a description to the failure message.

Reply:
@paulodamaso fixed